src/components/Organization.vue

```diff
@@ -35,16 +35,16 @@
 							:tab="myTab"
 							:active-tab="activeTab"
 							:depth="depth"
-							:disabled="i!==lastLevel"
+							:read-only="i!==lastLevel || readOnly"
 							v-bind="ui.props.referenceData"
 							:es-query-extra="getQueryExtraForLevel(i)"
 							:placeholder="getPlaceholderForLevel(i)"
 							:actions="actions"
 							:disable-internal-search="true"
 							:preserved-fields="hoistedFields"
-							:allowEmpty="false"
-							@changed="()=>handleChanged(i)"
-							@action="(action)=>handleAction(i, action)"
+							:allow-empty="false"
+							@changed="handleChanged(i)"
```

Comment:
was this on purpose?

Reply:
yes, in Vue you can pass parameters like that directly in the template without using an arrow function https://vuejs.org/v2/guide/events.html#Methods-in-Inline-Handlers